src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -757,6 +808,7 @@ public int WorkingSet
         {
             get
             {
+                CheckDisposed();
```

Comment:
The `CheckDisposed` check is in `EnsureState` already. No need to duplicate it on all these places.

Reply:
removed